features/step_definitions/basic_steps.rb

```diff
@@ -379,15 +381,15 @@
 end
 
 When /^I fill in "(.*?)" with "(.*?)" within the navbar$/ do |field, value|
-  within('#navbar') { fill_in(field, :with => value) }
+  within('#navbar') { fill_in(field, with: value) }
 end
 
 When /^I fill in "(.*?)" with "(.*?)" within the main body$/ do |field, value|
-  within('#main') { fill_in(field, :with => value) }
+  within('#main') { fill_in(field, with: value) }
 end
 
 Given /^I create "(.*?)" org$/ do |name|
-  page.driver.submit :post, "/organisations", :organisation => {:name => name}
+  page.driver.submit :post, "/organisations", organisation: {name: name}
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)